unimodule.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "heap",
```

Comment:
What does this file do?

Reply:
I'm not seeing it in the packaged file. `tar -ztvf heap-react-native-heap-0.21.0.tgz |grep uni`

Also, should we call it `react-native-heap` to match the SDK?